docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -0,0 +1,71 @@
+---
+---
+
+{% capture overview %}
+This page shows how to change reclaim policy of a Kubernetes Persistent Volume.
+{% endcapture %}
+
+{% capture prerequisites %}
+
+{% include task-tutorial-prereqs.md %}
+
+{% endcapture %}
+
+{% capture steps %}
+
+### Why change reclaim policy of a PersistentVolume
+
+`PersistentVolumes` can have various reclaim policies and the default
+policy of dynamically provisioned `PersistentVolumes` is "Delete".
+This means that a dynamically provisioned volume is automatically deleted
+when user deletes corresponding PeristentVolumeClaim. Such automatic
+behavior can be too dangerous if the volume contains precious data and it
```

Comment:
s/can be too dangerous/may be inappropriate/

s/and it might be useful to change the policy to "Retain"/. Instead using the "Retain" policy may be more appropriate/